packages/features/bookings/components/event-meta/AvailableEventLocations.tsx

```diff
@@ -98,9 +98,9 @@ export function AvailableEventLocations({ locations }: { locations: LocationObje
   return filteredLocations.length > 1 ? (
     <div className="flex flex-row items-center text-sm font-medium">
       <img
-        src="/map-pin.svg"
+        src="/map-pin-dark.svg"
         className={classNames("me-[10px] h-4 w-4 opacity-70 dark:invert")}
-        alt="map-pin"
+        alt="map-pin-dark"
```

Comment:
```suggestion
        alt="map-pin"
```

I don't think alt needs to have dark in it. It is read by accessibility tools and doesn't have any significance there